.github/workflows/deployments.yml

```diff
@@ -115,3 +115,26 @@ jobs:
           context: .
           file: ./docker/katana/Dockerfile
           platforms: linux/amd64,linux/arm64
+
+  build_and_push_deployer:
+    runs-on: ubuntu-latest
+    needs: deploy
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and push
+        uses: docker/build-push-action@v4
+        with:
+          push: true
+          tags: ghcr.io/kkrt-labs/kakarot/deployer:latest
+          context: .
+          file: ./docker/deployer/Dockerfile
```

Comment:
can we add `platforms: linux/amd64,linux/arm64` here? would be good to support both